src/_guides/language/effective-dart/design.md

```diff
@@ -1003,24 +1035,27 @@ The language specifies that this check is done automatically and your `==`
 method is called only if the right-hand side is not `null`.
 
 {:.good-style}
+<?code-excerpt "misc/lib/effective_dart/design_good.dart (eq-dont-check-for-null)" replace="/operator ==|hashCode/[!$&!]/g"?>
 {% prettify dart %}
 class Person {
   final String name;
+  // ...
+  @override
```

Comment:
Why add `@override`. The style guide does not recommend using `@override`. There is nothing "good" or "bad" about using it, so I don't think it should be added to a "good" section.

Reply:
I've been trying to avoid disabling any of the lint rules unless necessary.
Anyhow, I've removed `@override`s.